docs/src/gettingstarted.md

```diff
@@ -356,4 +358,4 @@ julia> Conda.add("tbb=2020.2", env; channel="conda-forge")
 
 If your issue isn't addressed here, or you're still having problems,
 please [open an issue](https://github.com/BioJulia/BiobakeryUtils.jl/issues/new/choose)
-or start a discussion over on [`Microbiome.jl`](https://github.com/BioJulia/Microbiome.jl/discussions/new).
\ No newline at end of file
+or start a discussion over on [`Microbiome.jl`](https://github.com/BioJulia/Microbiome.jl/discussions/new).
```

Comment:
Damnit - this should be `.../EcoJulia/Microbiome.jl/discussions/new` - my project-wide grep seems to have failed again. In this case, the link should still work because github redirects